test/types/string/parIter/zipUTF8.chpl

```diff
@@ -0,0 +1,9 @@
+var s = "çççççççççççççççççç";
+
+var a: [0..<s.size] string;
+
+forall (elem, char) in zip(a, s) {
+  elem = char;
+}
```

Comment:
Should we add a version where `s` leads as well?  I guess the `parIter/zip.chpl` test covers this case, though if it started failing, I think I'd much rather debug this test than that one...